mlir/lib/Conversion/MemRefToSPIRV/MemRefToSPIRV.cpp

```diff
@@ -508,11 +568,15 @@ IntLoadOpPattern::matchAndRewrite(memref::LoadOp loadOp, OpAdaptor adaptor,
   assert(accessChainOp.getIndices().size() == 2);
   Value adjustedPtr = adjustAccessChainForBitwidth(typeConverter, accessChainOp,
                                                    srcBits, dstBits, rewriter);
-  Value spvLoadOp = rewriter.create<spirv::LoadOp>(
-      loc, dstType, adjustedPtr,
-      loadOp->getAttrOfType<spirv::MemoryAccessAttr>(
-          spirv::attributeName<spirv::MemoryAccess>()),
-      loadOp->getAttrOfType<IntegerAttr>("alignment"));
```

Comment:
I'm not familiar with this path; if `PhysicalStorageBuffer` is not the storage class, is it possible to have a memory access attribute without an alignment or vice-versa? The new logic looks to return two empty attributes in this case.

Reply:
It's a storage class btw

https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/KHR/SPV_KHR_physical_storage_buffer.asciidoc

Reply:
Both must be present at the same time in SPIR-V